azure-pipelines.yml

```
@@ -0,0 +1,62 @@
+trigger:
+  branches:
+    include:
+    - main
+    - develop
+pr:
+  branches:
+    include:
+    - '*' 
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- checkout: self
+  clean: true
+
+- task: UseDotNet@2
+  displayName: Set .NET SDK to 5.x
+  inputs:
+    packageType: 'sdk'
+    version: '5.x'
+
+- task: SonarCloudPrepare@1
+  displayName: Prepare Sonarcloud
+  inputs:
+    SonarCloud: 'Sonarcloud'
+    organization: 'dept'
+    scannerMode: 'MSBuild'
+    projectKey: 'dtnl-livelight-website'
+    projectName: 'dtnl-livelight-website'
+
+- task: DotNetCoreCLI@2
+  displayName: 'Dotnet: Restore'
+  inputs:
+    command: 'restore'
+    projects: 'DTNL.LL.Website/DTNL.LL.Website.csproj'
+
+- task: DotNetCoreCLI@2
+  displayName: 'Dotnet: Build'
+  inputs:
+    command: 'build'
+    projects: 'DTNL.LL.Website/DTNL.LL.Website.csproj'
+
+- task: DotNetCoreCLI@2
+  displayName: 'Dotnet: Publish'
+  inputs:
+    command: 'publish'
+    publishWebProjects: false
+    projects: 'DTNL.LL.Website/DTNL.LL.Website.csproj'
+    arguments: '--configuration Release --output $(build.artifactstagingdirectory)'
+    modifyOutputPath: false
+
+- task: SonarCloudAnalyze@1
+  displayName: Run Sonarcloud analyzer
+
+- task: PublishBuildArtifacts@1
+  displayName: Publish artifacts
+  inputs:
+    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
+    ArtifactName: 'backend'
+    publishLocation: 'Container'
```

This task is deprecated please use the `PublishPipelineArtifact` task instead:
```suggestion
- publish: '$(Build.ArtifactStagingDirectory)'
  artifact: 'backend'
```

---

Thanks. Had no idea that the task was deprecated. Updated it to the new task